Makefile

```diff
@@ -0,0 +1,50 @@
+ROOT := $(shell pwd)
+
+# The default is to download the official release binary
+# for x86_64-unknown-linux-gnu. If that default is unsuitable,
+# build your own binary and copy or symlink it here, or
+# override the variable with "make MDBOOK_BINARY=...".
+MDBOOK_BINARY = $(ROOT)/mdbook
+
+# The architecture can be changed. Only Linux .tar.gz files
+# are currently supported, though. See https://github.com/rust-lang-nursery/mdBook/releases
+# for available architectures.
+MDBOOK_ARCH = x86_64-unknown-linux-gnu
```

Comment:
How about using the docker image that I built `chengpan/mdbook:0.2.1` so that we don't need to worry about installing mdbook and cross platform issue? I will be happy to transfer it to some official repo if people has concerns it is a personal repo.

Reply:
I did indeed have concerns about the maintenance of that docker image. After having looked at how difficult (or rather, easy) it is to just download the pre-build mdbook, that's the approach that I prefer personally.

Reply:
Another reason is that for Travis CI, using Docker is just plain overkill.